server.js

```diff
@@ -49,15 +49,27 @@ app.post('/api/register', async (req, res) => {
   }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
-
 // app.post('/api/achievement', async (req, res) => {
 //   console.log("It's alive!");
 //   res.status(200).json({ message: 'Achievement successful' });
 // });
 
+app.post('/api/attempt', async (req, res) => {
+  try {
+    const { characters_attempted, characters_missed, wpm, email } = req.body;
+    // console.log({ characters_attempted, characters_missed, wpm, email })
+    const getUserId = await getUserIdByEmail(email);
+    const result = insertAttempt(getUserId, characters_attempted, characters_missed, wpm);
+    res.status(200).json(result);
+  } catch (err) {
+    console.error("Attempt insertion error:", err);
+    res.status(500).json({ error: "Can't insert attempt. " + err.message });
+  }
+});
+
+app.listen(port, () => {
+  console.log(`Server is running on port ${port}`);
```

Comment:
Related to previous nit about adding square brackets.

```suggestion
  console.log(`Server is running on port [${port}]`);
```